lib/airbrakex/notifier.ex

```diff
@@ -3,6 +3,7 @@ defmodule Airbrakex.Notifier do
 
   @request_headers [{"Content-Type", "application/json"}]
   @default_endpoint "http://collect.airbrake.io"
+  @env Mix.env
```

Comment:
Currently I'm attending ElixirConfEU, so I don't have a chance to make this change. I'll do this change hopefully tomorrow.

Sent from my iPhone

> On 12 May 2016, at 09:10, Michał Kalbarczyk <notifications@github.com> wrote:
> 
> In lib/airbrakex/notifier.ex:
> 
> > @@ -3,6 +3,7 @@ defmodule Airbrakex.Notifier do
> >  
> >    @request_headers [{"Content-Type", "application/json"}]
> >    @default_endpoint "http://collect.airbrake.io"
> > +  @env Mix.env
> Can you change @env to @default_env to be more verbose ?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>